explain.jsx

```diff
@@ -49,27 +50,37 @@ App.Explain = React.createClass({
               madeByPlural = "furnaces";
               assemblerPartOne = (<span>Your chosen furnace has a crafting speed of { bmono(this.props.options.smeltlvl) }. This means it will take {decimalNumber(recipe.assemblyTime)} seconds to smelt { wholeNumber(recipe.outputs)} { bmono(recipe.name) }.</span>);
             } else if (recipe.category == "chemistry") {
-              madeBySingular = "furnace";
+              madeBySingular = "chemplant";
               madeByPlural = "chemplants";
               assemblerPartOne = (<span>Chemplants have a crafting speed of { bmono(1.25) }. This means it will take {decimalNumber(recipe.assemblyTime)} seconds to create { wholeNumber(recipe.outputs)} { bmono(recipe.name) }.</span>);
             }
-
+            if (recipe.type == "fluid")
+            {
+              madeUnits = "units"
+            }
+            
+            var assemb = null;
```

Comment:
I'm going to change this to "assemblyLinesExplanation" just for clarity after merging.